lib/src/generator/html_generator.dart

```diff
@@ -37,6 +37,52 @@ class HtmlGeneratorBackend extends GeneratorBackendBase {
   HtmlGeneratorBackend(
       super.options, super.templates, super.writer, super.resourceProvider);
 
+  @override
+  void generateClass(PackageGraph packageGraph, Library library, Class clazz) {
```

Comment:
Each of these which used to just render one HTML file must now render two, with the addition of their sidebar file, so they are overridden here, adding a lot of repeated boilerplate.